vendors/package.json

```diff
@@ -33,7 +33,7 @@
     "ng-focus-if": "~1.0.7",
     "ng-tags-input": "~3.2.0",
     "ng-wig": "~3.0.19",
-    "openlayers-ext": "~3.1.5",
+    "ol-ext": "~3.2.3",
```

Comment:
ol-ext was causing build failures, updating fixed this (thanks @axelstudios !) -- note openlayers was a dependency of openlayers-ext, which is a deprecated version

```
#15 86.10 npm WARN deprecated openlayers@4.6.5: Please use the ol package instead
#15 119.8 npm ERR! code 1
#15 119.8 npm ERR! path /seed/vendors/node_modules/closure-util
#15 119.8 npm ERR! command failed
#15 119.8 npm ERR! command sh -c node ./bin/closure-util.js update
#15 119.8 npm ERR! info install Downloading http://github.com/google/closure-library/archive/v20170910.zip
#15 119.8 npm ERR! ERR! closure-util EEXIST: file already exists, mkdir '/seed/vendors/node_modules/closure-util/.deps/library/947e46b02fd5e472c2daa86669c09303ca3e7033/closure-library-20170910/third_party/closure/goog/caja/string'
#15 120.0 
#15 120.0 npm ERR! A complete log of this run can be found in:
#15 120.0 npm ERR!     /root/.npm/_logs/2021-07-06T15_52_04_273Z-debug.log
#15 120.1 npm ERR! code 1
#15 120.1 npm ERR! path /seed
#15 120.1 npm ERR! command failed
#15 120.1 npm ERR! command sh -c cd vendors && npm install
#15 120.2 
#15 120.2 npm ERR! A complete log of this run can be found in:
#15 120.2 npm ERR!     /root/.npm/_logs/2021-07-06T15_52_04_585Z-debug.log
```